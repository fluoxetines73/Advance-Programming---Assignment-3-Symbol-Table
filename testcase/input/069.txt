INSERT g_1 number
INSERT g_1 string
PRINT
INSERT a string
LOOKUP a
PRINT
END
BEGIN
BEGIN
BEGIN
INSERT x string
INSERT c number random
ASSIGN x '@stArt'
ASSIGN x 'Test123'
INSERT x string
LOOKUP x
END
END
PRINT
UWAAAA