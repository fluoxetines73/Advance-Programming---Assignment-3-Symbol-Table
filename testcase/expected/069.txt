Redeclared: INSERT g_1 string